src/Components/SmartComponents/Modals/BusinessRiskModal.test.js

```diff
@@ -65,4 +84,46 @@ describe('BusinessRiskModal', () => {
             cve: ['SOME CVE1', 'SOME CVE2', 'SOME CVE3', 'SOME CVE4']
         });
     });
+    it('Should call updateRef with goToFirstPage', async () => {
+        const testContext = { ...mockContext, cves: { meta: { page: 2 }}};
+        const cves = [
+            { id: 'SOME CVE1', business_risk_id: '3' },
+            { id: 'SOME CVE2', business_risk_id: '2' },
+            { id: 'SOME CVE3', business_risk_id: '1' },
+            { id: 'SOME CVE4', business_risk_id: '0' }
+        ];
+        render(
+            <TestWrapper store={ store }>
+                <CVETableContext.Provider value={testContext}>
+                    <BusinessRiskModal open cves={cves} intl={intl} goToFirstPage={true} />
+                </CVETableContext.Provider>
+            </TestWrapper>
+        );
+
+        fireEvent.click(screen.getByRole('radio', { name: 'Critical' }));
+        fireEvent.change(screen.getByRole('textbox', { name: /justification/i }), { target: { value: 'Test' } });
+        fireEvent.click(screen.getByRole('button', { name: 'Save' }));
+        await waitFor(() => expect(updateRef).toHaveBeenCalledWith({ page: 1 }, mockContext.params, mockContext.methods.apply));
```

Comment:
Nitpick: if we insert some params into `mockContext.params` instead of an empty object, the test would be more robust.